src/gui/Src/Gui/CPUDisassembly.cpp

```diff
@@ -1520,6 +1522,39 @@ void CPUDisassembly::copySelectionToFileSlot(bool copyBytes)
     }
 }
 
+void CPUDisassembly::copySelectionBytesSlot()

```

Comment:
Thanks for your contribution! Because we're in the GUI code directly there is a slightly different framework for working with the data there. In this case the code can be simplified:

```cpp
QString clipboard = "";
    prepareDataRange(getSelectionStart(), getSelectionEnd(), [&](int i, const Instruction_t & inst)
    {
        if(i)
            clipboard += "\r\n";
        clipboard += "TODO: convert inst.dump to hex";
        return true;
    });
    Bridge::CopyToClipboard(clipboard);
```

Reply:
Oh, thank you for the guidance! I was looking for an easier way to implement this than I did, but I couldn’t figure it out. Of course, I’ll update it soon.